sql/files/defaultdata/r/run

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh
+
+# R compile wrapper-script for 'compile.sh'.
+# See that script for syntax and more info.
+#
+# This script does not actually "compile" the source, but writes a
+# shell script that will function as the executable: when called, it
+# will execute the source with the correct interpreter syntax, thus
+# allowing this interpreted source to be used transparantly as if it
+# was compiled to a standalone binary.
+#
+# NOTICE: when this compiler script is used with USE_CHROOT on, the
+# chroot requires r to be installed
+
+DEST="$1" ; shift
+MEMLIMIT="$1" ; shift
+MAINSOURCE="${ENTRY_POINT:-$1}"
+
+# Report the entry point, so it can be saved, e.g. for later replay:
+if [ -z "$ENTRY_POINT" ]; then
+    echo "Info: detected entry_point: $MAINCLASS"
+fi
+
+# Check syntax
+Rscript "$@"
```

Comment:
won't it hang and wait for input if there is reading from stdin inside R script? I don't know of any pycompile alternative on R - there is lintr but it's external package not available in default install. Maybe skip this like in for example awk language?

Reply:
I was thinking of something like:
```
R -e "install.packages('lintr')"
lint_command="library(lintr); lint('$@')"
R -e "$lint_command"
```
but for a reason I don't know it's not working as expected, in particular it doesn't return non-zero exit code (but only prints result to output) while error is detected. Yes, this check should be removed.